ols/app/endpoints/ols.py

```diff
@@ -788,10 +808,7 @@ def store_transcript(
         "llm_response": response,
         "rag_chunks": [dataclasses.asdict(rag_chunk) for rag_chunk in rag_chunks],
         "truncated": truncated,
-        "tool_calls": [
-            [dataclasses.asdict(tool_call) for tool_call in step_tool_calls]
-            for step_tool_calls in tool_calls
-        ],
+        "tool_calls": merge_tools_info(tools_calls, tools_results),
```

Comment:
To avoid handling transcripts schema change in reporting (we already have archives with "tool_calls"), I merge the tools and execution here.